### ISSUE TITLE
Backup stops after folder exclusion

### ISSUE THREAD

**Issue by arjen-w** [jcwillox/hass-auto-backup#63]
After upgrading to HA version 2022.7.0 my backups are incomplete.
It seems like the job does not include any add-ons when a folder exclusion is  configured.

My task config:

```yaml
service: auto_backup.backup_full
data:
  name: 'DailyBackup: {{ now().strftime(''%A, %B %-d, %Y'') }}'
  keep_days: 2
  exclude:
    folders:
      - share
```

This always worked fine until the upgrade of HA.
A task without the excluded folder is working fine though, but as my share folder contains video recordings that I don't need to be backed-up, I want to exclude that folder.

The supervisor logs does not show any error.

**Comment by jcwillox** [comment]
Well, I figured it out, this is also related to #62. It turns out it's not a bug I introduced but actually a silent change to the Supervisor API, which removed the `installed` property from the addons endpoint which now only includes installed addons.

This issue should be fixed in release [1.1.1](https://github.com/jcwillox/hass-auto-backup/releases/tag/1.1.1).

**Comment by arjen-w** [comment]
It seems solved in 1.1.1 indeed! thank you for the fast actions!
